tests/test_cookiecutter_django_app.py

```diff
@@ -4,15 +4,15 @@
 
 import logging
 import logging.config
-import os
 import re
 from pathlib import Path
 
 import pytest
 import sh
 
 from .bake import bake_in_temp_dir
-from .venv import run_in_virtualenv
+from .common_tests import *  # pylint: disable=wildcard-import
```

Comment:
Oh cool, that works? I guess that works!

Reply:
I had to experiment to see if it did.  Maybe I'll add a comment explaining what is happening, since it might not be obvious that the test will be run once for each file it's imported into.